server/server.js

```diff
@@ -0,0 +1,107 @@
+const SERVER_PORT = process.env.PORT || 4000;
+const express = require("express");
+const cors = require("cors");
+const { Pool } = require('pg');
+
+// get the full list of albums
+const albumsData = require("./albums");
```

Comment:
please remove this and all the other stuff related to albums.js and albums.js itself.